bbq/commons/types.go

```diff
@@ -19,14 +19,33 @@
 package commons
 
 import (
-	"bytes"
-
 	"github.com/onflow/cadence/common"
 	"github.com/onflow/cadence/interpreter"
 	"github.com/onflow/cadence/sema"
 )
 
-func TypeQualifiedName(typeName, functionName string) string {
+var BuiltinTypes = common.Concat[sema.Type](
```

Comment:
Moved it here from `bbq/compiler/native_functions.go` , so that it can be used in both compiler and vm for native functions registering.